perimeter.js

```diff
@@ -1,10 +1,3 @@
-/**
- * @param {number} length - vertical length
- * @param {number} width - horizontal width
- * @return {number} - returns the perimeter 
-   of given length and width
- */
-
 function findPerimeter(length, width) {

```

Comment:
/**
 * calculate perimeter of the rectangle
 * @param {number} length - length of the rectangle
 * @param {number} width - width of the rectangle
 * @returns {number} - returns perimeter of the rectangle
 */